api/validators/MerchStoreRequests.ts

```diff
@@ -16,13 +16,17 @@ import {
   MerchItemEdit as IMerchItemEdit,
   MerchItemOption as IMerchItemOption,
   MerchItemOptionEdit as IMerchItemOptionEdit,
+  MerchItemOptionMetadata as IMerchItemOptionMetadata,
 } from '../../types';
 
 export class MerchCollection implements IMerchCollection {
   @IsDefined()
   @IsNotEmpty()
   title: string;
 
+  @Allow()
+  themeColorHex?: string;
```

Comment:
Should we validate that this is actually a hexadecimal value with expected length? Or just leave that to the FE like `position`. I think we should.

Reply:
Agreed